src/test/kotlin/org/rust/ide/docs/RsQuickDocumentationTest.kt

```diff
@@ -743,6 +743,18 @@ class RsQuickDocumentationTest : RsDocumentationProviderTest() {
         <div class='definition'><pre>variable <b>x</b>: S&lt;i32&gt;</pre></div>
     """)
 
+    fun `test variable type alias`() = doTest("""
+        struct S;
+        type T = S;
+
+        fn main() {
+            let x: T = T;
```

Comment:
```suggestion
            let x: T = S;
```

Otherwise, it doesn't compile